docs/releasing/testing-and-linting.md

```diff
@@ -2,55 +2,88 @@
 
 The CI lints SASS and JavaScript, runs unit and functional tests with Node, and generates snapshots for [visual regression testing](https://www.browserstack.com/percy/visual-testing).
 
-## Running all tests locally
+## Testing terminology
+
+We use different types of tests to check different areas of our code are working as expected.
+
+Unit tests are small, modular tests that verify a "unit" of code. We write unit tests to check our JavaScript logic, particularly 'background logic' that doesn't heavily rely on DOM interaction (where functional tests may be better suited).
+
+Functional tests verify the output of an action and do not check the intermediate states of the system when performing that action. We write functional tests to check component interactions have the expected results, so we also refer to these as component tests. We also write functional tests to check that our Nunjucks code outputs expected HTML, and we tend to refer to these as our Nunjucks tests.
```

Comment:
Suggest removing 'tend to'. If we use more than one terminology for the test, we should say under what circumstances we call them Nunjucks tests or [some other tests] under other circumstances.